ecommerce/constants.py

```diff
@@ -0,0 +1,24 @@
+"""Constants for ecommerce."""
+
+DISCOUNT_TYPE_PERCENT_OFF = "percentoff"
+DISCOUNT_TYPE_DOLLARS_OFF = "dollarsoff"
+DISCOUNT_TYPE_FIXED_PRICE = "fixedprice"
+
+ALL_DISCOUNT_TYPES = [
+    DISCOUNT_TYPE_PERCENT_OFF,
+    DISCOUNT_TYPE_DOLLARS_OFF,
+    DISCOUNT_TYPE_FIXED_PRICE,
+]
+DISCOUNT_TYPES = list(zip(ALL_DISCOUNT_TYPES, ALL_DISCOUNT_TYPES))
+
+REDEMPTION_TYPE_ONE_TIME = "onetime"
+REDEMPTION_TYPE_ONE_TIME_PER_USER = "onetimeperuser"
+REDEMPTION_TYPE_UNLIMITED = "unlimited"
```

Comment:
Nit: could you dash-delimit  the words in the constant values so it's easier to read?

Reply:
Should be fixed in ad31885b13c9acb18693d2edf90d1778735ecdf9